src/Helpers/tvInformation.js

```diff
@@ -0,0 +1,68 @@
+import {inventory} from "../constants/inventory.js";
+
+// Naam, merk en type van de tv
+function tvName(tv) {
+
+    const name = tv.name;
+    const type = tv.type;
+    const brand = tv.brand;
+
+    return brand + " " + type + " - " + name;
```

Comment:
Vind ik zelf altijd makkelijker te lezen en aan te passen
```suggestion
    return `${brand} ${type} - ${name}`;
```